flavors/standard-Exasol-8-cuda-ml/flavor_base/flavor_base_deps/packages/conda_channels

```diff
@@ -0,0 +1,3 @@
+conda-forge
+rapidsai
```

Comment:
Not needed, if we don't install rapidsai

Reply:
True. Removed it